packages/x-date-pickers/src/DesktopNextTimePicker/DesktopNextTimePicker.tsx

```diff
@@ -34,8 +34,6 @@ const DesktopNextTimePicker = React.forwardRef(function DesktopNextTimePicker<TD
     ...defaultizedProps.viewRenderers,
   };
 
-  const slots = defaultizedProps.slots;
```

Comment:
Some components were defining those variables.
Some components were spreading them from `defaultizedProps`
I unified it :+1: